core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -84,19 +85,60 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
     }
 
     /**
-     * Returns a {@link Backoff} that provides a delay that increases using
+     * Returns a {@link Backoff} that provides a jitter value for the backoff delay using
```

Comment:
How about:

- provides a jitter value for the backoff delay -> adds a random jitter value to the original delay